packages/teams-js/rollup.config.mjs

```diff
@@ -7,7 +7,7 @@ import replace from '@rollup/plugin-replace';
 import terser from '@rollup/plugin-terser';
 import typescript from '@rollup/plugin-typescript';
 import { readFileSync } from 'fs';
-import nodePolyfills from 'rollup-plugin-node-polyfills';
+import nodePolyfills from 'rollup-plugin-polyfill-node';
```

Comment:
Is the global polyfill included in this upgraded package?

Reply:
Yes, due to some differences in how these two plugins function, using the new plugin will result in the global polyfill being included in the bundle.